bindings/generate_bindings.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import subprocess
+from pathlib import Path
+
+def uniffi_gen(executable: str, *args):
+    args = list(str(a) for a in args)
+    uniffi = [executable]
+    uniffi.extend(args)
+    print(f'Running {" ".join(uniffi)}')
+    subprocess.run(uniffi, check=True)
```

Comment:
Minor tip: I think it can look like this:
```suggestion
def uniffi_gen(executable: str, *args):
    print(f'Running {executable} {" ".join(map(str, args))}')
    subprocess.run([executable, *map(str, args)], check=True)
```

Reply:
Uu, that's nice. Thank you!